MapboxCoreNavigation.podspec

```
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 123.0"
+  s.dependency "MapboxNavigationNative", "~> 123.1.0"
```

I think it'd be better to use:
```suggestion
  s.dependency "MapboxNavigationNative", "~> 123.1"
```

---

Thanks for the suggestion, I'll update that.